app/src/main/res/values/strings.xml

```diff
@@ -326,7 +327,8 @@
   <string name="profile_reset_pin_error_user_pin_length">Your PIN should be 3 digits long.</string>
   <string name="profile_reset_pin_error_admin_pin_length">Your PIN should be 5 digits long.</string>
   <string name="add_profile_create_a_3_digit_pin">Create a 3-Digit PIN</string>
-  <string name="add_profile_required">*Required</string>
+  <string name="add_profile_required">Required</string>
+  <string name="add_profile_required_star">*</string>
```

Comment:
Remove `add_profile_required_star ` as this is not required.

Reply:
Done